boot.php

```diff
@@ -15,9 +20,9 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+$filename = getcwd().'/'.$argv[1];
```

Comment:
@mauri870: How symfony/console will solve this problem? We can just check if the $argv[1] starts with DIRECTORY_SEPARATOR (Linux) or '(char):' (Windows), and it will be an absolute path.